CMakeLists.txt

```diff
@@ -60,7 +66,8 @@ endmacro()
 #
 # Cereal
 #
-find_path(CEREAL_INCLUDE_DIR cereal/cereal.hpp)
+#find_path(CEREAL_INCLUDE_DIR cereal/cereal.hpp)
+set(CEREAL_INCLUDE_DIR /g/g92/ssriniv/packages/cereal/include)
```

Comment:
Just noticed my local build got pushed. I forgot to add CMake to git ignore. I will take care of that